### ISSUE TITLE
Showing username or avatar

### ISSUE THREAD

**Issue by vanthao03596** [antoine92190/vue-advanced-chat#144]
## Is your feature request related to a problem? Please describe

It would be great if we can show username or avatar of sender

## Describe the solution you'd like

Maybe avatar image like fb messengers

**Comment by antoine92190** [comment]
Yes, that can be a nice addition. Will think about it and keep in touch

**Comment by antoine92190** [comment]
Feature added in https://github.com/antoine92190/vue-advanced-chat/commit/7d8bdec8b2b76b4e66d000731f4443c38ae77b9b
To show avatar on a message, you need to pass `avatar` in `messages`

**Comment by antoine92190** [comment]
You cannot a the moment. I can add it to show current user avatar in case one is passed in the messages prop.
